dir.props

```diff
@@ -80,6 +72,15 @@
     <Platform Condition="'$(Platform)' == 'amd64'">x64</Platform>
   </PropertyGroup>
 
+  <!--
+     Switching to the .NET Core version of the BuildTools tasks seems to break numerous scenarios, such as VS intellisense and resource designer
+     as well as runnning the build on mono. Until we can get these sorted out we will continue using the .NET 4.5 version of the tasks.
+     -->
+  <PropertyGroup>
+    <BuildToolsTargets45 Condition="'$(BuildOS)' != 'Windows_NT'">false</BuildToolsTargets45>
```

Comment:
I don't think so because we explicitly declare build OS. If we have BuildOS as "AnyOS" it means error. Moreover "AnyOS" means "Not Windows_NT" which also correct and we should use cross-platform version of framework.